### ISSUE TITLE
Cannot find module 'typescript'

### ISSUE THREAD

**Issue by moonray** [svipas/vscode-prettier-plus#13]
This just randomly started happening; it has worked for ages.

```
/Users/bala/Develop/feathr-extender/packages/components/src/Button/ButtonValid.tsx (12/17/2019, 9:57:25 AM):
------------------------------------------------------------------------------------------------------------
Using prettier-tslint@0.4.2 with Prettier config to format code with typescript parser for typescriptreact language.

/Users/bala/Develop/feathr-extender/packages/components/src/Button/ButtonValid.tsx (12/17/2019, 9:57:25 AM):
------------------------------------------------------------------------------------------------------------
Cannot find module 'typescript'
Require stack:
- /Users/bala/.vscode/extensions/svipas.prettier-plus-3.6.0/node_modules/prettier-tslint/dist/create-program.js
- /Users/bala/.vscode/extensions/svipas.prettier-plus-3.6.0/node_modules/prettier-tslint/dist/run-tslint.js
- /Users/bala/.vscode/extensions/svipas.prettier-plus-3.6.0/node_modules/prettier-tslint/dist/fix.js
- /Users/bala/.vscode/extensions/svipas.prettier-plus-3.6.0/node_modules/prettier-tslint/dist/cli.js
- /Users/bala/.vscode/extensions/svipas.prettier-plus-3.6.0/node_modules/prettier-tslint/dist/index.js
- /Users/bala/.vscode/extensions/svipas.prettier-plus-3.6.0/dist/main.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-fork.js
```

Any ideas?

**Comment by svipas** [comment]
Hmm, very weird, it's coming from `prettier-tslint`, could you share the Prettier config, tslint config and file content? I would like to try it on my machine maybe I could repro.

**Comment by moonray** [comment]
.prettierrc:
```json
{
  "printWidth": 100,
  "semi": true,
  "singleQuote": true,
  "trailingComma": "all"
}
```

tsconfig.json:
```json
{
  "compilerOptions": {
    "baseUrl": "./",
    "outDir": "./dist/",
    "allowSyntheticDefaultImports": true,
    "allowJs": true,
    "checkJs": false,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "forceConsistentCasingInFileNames": true,
    "importHelpers": true,
    "noEmitHelpers": true,
    "jsx": "preserve",
    "lib": ["dom", "es2016", "es2017"],
    "target": "esnext",
    "module": "esnext",
    "moduleResolution": "node",
    "noEmitOnError": true,
    "noFallthroughCasesInSwitch": true,
    "noImplicitAny": true,
    "noImplicitReturns": true,
    "noImplicitThis": true,
    "noUnusedLocals": true,
    "strict": true,
    "pretty": true,
    "sourceMap": true,
    "downlevelIteration": true,
    "esModuleInterop": true,
    "removeComments": false
  },
  "exclude": ["./node_modules", "./dist", "./coverage", "**/node_modules/*", "**/dist/*"]
}
```

tslint.json:
```json
{
  "rules": {
    "eofline": true,
    "forin": false,
    "jsx-no-lambda": false,
    "max-classes-per-file": false,
    "object-literal-sort-keys": false,
    "ordered-imports": {
      "options": {
        "import-sources-order": "case-insensitive",
        "grouped-imports": true,
        "groups": [
          {
            "match": "^(babel-polyfill|whatwg-fetch)$",
            "order": 0
          },
          {
            "match": "^normalize.css$",
            "order": 1
          },
          {
            "match": "\\.css$",
            "order": 30
          },
          {
            "match": "\\.json$",
            "order": 40
          },
          {
            "match": "\\.(gif|jpg|png)$",
            "order": 50
          },
          {
            "match": "^(src|\\.\\.|\\.)/",
            "order": 20
          },
          {
            "match": ".*",
            "order": 10
          }
        ],
        "module-source-path": "full",
        "named-imports-order": "case-insensitive"
      }
    },
    "quotemark": [true, "single", "jsx-double", "avoid-escape"],
    "semicolon": [true, "always", "strict-bound-class-methods"],
    "variable-name": [true, "allow-snake-case", "allow-pascal-case"]
  },
  "extends": ["tslint:recommended", "tslint-react", "tslint-config-prettier"],
  "linterOptions": {
    "exclude": [
      "webpack.config.js",
      "webpack.base.config.js",
      "webpack.production.config.js",
      "*.json",
      "*.md",
      "**/node_modules/**/*"
    ]
  }
}
```

```typescript
import React from 'react';

import Tooltip from '../Tooltip';
import Button, { IButtonProps } from './Button';

interface IButtonValidProps extends Omit<IButtonProps, 'disabled' | 'type'> {
  children: React.ReactNode;
  errors: string[];
  onClick: () => void;
}

function ButtonValid({ children, errors, ...buttonProps }: IButtonValidProps) {
  const isValid = errors.length === 0;

  const button = (
    <Button {...buttonProps} type={'primary'} disabled={!isValid}>
      {children}
    </Button>
  );

  return isValid ? (
    button
  ) : (
    <Tooltip title={errors.reduce((acc, error) => (acc += ` ${error}`), '')}>{button}</Tooltip>
  );
}

export default ButtonValid;
```

**Comment by moonray** [comment]
My version of vscode is  1.41.0
Commit: 9579eda04fdb3a9bba2750f15193e5fafe16b959

**Comment by svipas** [comment]
Okay I know the problem. It doesn't include `typescript` dependency into `node_modules`. Seems `vsce` contains `--yarn` flag which fixes this, so I'm gonna republish extension. Good catch, seems I need to do that for every extension I have since I'm using Yarn instead of npm.

**Comment by svipas** [comment]
Should be fixed in `3.6.1` version.

**Comment by moonray** [comment]
Thanks so much! All working once more.
